pkg/batch/cmd/server/server.go

```diff
@@ -267,6 +274,17 @@ func (s *server) Run(ctx context.Context, metrics metrics.Metrics, logger *zap.L
 		executor.WithLogger(logger),
 	)
 
+	experimentCalculatorClient, err := experimentcalculatorclient.NewClient(*s.experimentCalculatorService, *s.certPath,
```

Comment:
Please close the experimentalCalculatorClient in the defer function.

Reply:
Done!